server/tailsql/static/script.js

```diff
@@ -7,18 +9,60 @@
     const origin = document.location.origin;
     const sources = document.getElementById('sources');
     const body = document.getElementById('tsql');
+    const velo = 8, delay = 100, runChance = 0.01;
+    let hasRun = false;
+
+    const param = new Params(256, 256, 8, 8);
+    const aRunRight = new Loop(velo, 0, 5, [5,1,2,3]);
+    const aRunLeft = new Loop(-velo, 0, 6, [5,1,2,3]);
 
     function hasQuery() {
         return query.value.trim() != "";
     }
 
+    function shouldSquirrel() {
+        return !hasRun &&
+            query.value.trim().toLowerCase().includes("squirrel") &&
+            Math.random() < runChance;
+    }
+
+    function maybeRunSquirrel() {
+        if (!shouldSquirrel()) {
+            return;
+        }
+        // Squirrel art from:
+        //   http://saralara93.blogspot.com/2014/03/concept-art-part-3-squirrel.html
+
+        const nut = document.createElement("div");
```

Comment:
You could put this element in the HTML instead and just have it always be there but invisible, so that the whole interface doesn't shift when the squirrel appears.

Reply:
Hmm, I had originally added it there with `display:none`, but it still moves the item above. Probably I should move the "version" stamp inside the container so the squirrel runs over top of it instead of hoisting it.